default/stringtables/en.txt

```diff
@@ -19542,27 +19573,68 @@ Willow'''
 
 # Newline separated list of mineral-based ship names
 SHIP_NAME_STONE
-'''Amphibolite
+'''Adularia
+Agate
+Amber
+Amphibolite
 Andesite
+Anhydrite
+Asphalt
 Basalt
+Borax
+Chalcedony
 Chalk
+Chrysoberyl
 Clay
+Concrete
+Corundum
+Diamond
+Diopside
 Dolomite
+Emerald
+Feldspar
+Ferrite
 Flint
 Gabbro
 Gneiss
 Granite
+Haematite
+Hornblende
+Jasper
+Jett
 Komatite
+Lapislazuli
 Limestone
+Malachite
 Marble
+Melilith
+Menhir
+Nugget
+Obelisk
 Obsidian
+Onyx
+Opal
+Orthoclase
+Pebble
 Pegmatite
+Petrogenesis
 Pumice
+Pyrite
+Pyroxene
+Quartz
 Quartzite
+Rock
+Ruby
 Sandstone
+Saphire
```

Comment:
Sapphire I think?

Reply:
maybe. but is a name, so who cares ^^ well i can change or remove it if you want

Reply:
if you're editing anyway, please add a `p`